README.md

```diff
@@ -76,7 +76,7 @@ Download
 The project is available on jCenter. In your app build.gradle (or explicit module) you must add this:
 ```
 dependencies {
-  compile 'com.ezhome:rxfirebase:2.0.2'
+  compile 'com.ezhome:rxfirebase:2.2.2'
```

Comment:
you added wrong version dude? you prepared 2.2.0 not 2.2.2

Reply:
Yes, I know. I already push fix. https://github.com/ezhome/Android-RxFirebase/pull/18